Dockerfile

```
@@ -12,6 +12,22 @@ RUN set -x \
 	&& adduser -u 1000 -D -S -G bungeecord bungeecord \
 	&& addgroup bungeecord wheel
 
+# hook into docker BuildKit --platform support
+# see https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope
+ARG TARGETOS=linux
+ARG TARGETARCH=amd64
+ARG TARGETVARIANT=""
+
+ARG EASY_ADD_VER=0.7.1
+ADD https://github.com/itzg/easy-add/releases/download/${EASY_ADD_VER}/easy-add_${TARGETOS}_${TARGETARCH}${TARGETVARIANT} /usr/bin/easy-add
+RUN chmod +x /usr/bin/easy-add
+
+RUN easy-add --var os=${TARGETOS} --var arch=${TARGETARCH}${TARGETVARIANT} \
+ --var version=0.6.0 --var app=mc-monitor --file {{.app}} \
+ --from https://github.com/itzg/{{.app}}/releases/download/{{.version}}/{{.app}}_{{.version}}_{{.os}}_{{.arch}}.tar.gz
+COPY health.sh /
+
 CMD ["/usr/bin/run-bungeecord.sh"]
+HEALTHCHECK --start-period=10s CMD /health.sh
```

Looks like `SERVER_PORT` needs to be declared with a default since `health.sh` needs it:
```suggestion
ENV SERVER_PORT=25577
HEALTHCHECK --start-period=10s CMD /health.sh
```

---

Oh yes, i have missing that